### ISSUE TITLE
Failing test on non-english culture

### ISSUE THREAD

**Issue by khellang** [NuGet/NuGetGallery#5660]
I see a failing test when running them on a machine with Norwegian culture (which uses comma instead of dot for decimal separators):

```
NuGetGallery.Auditing.ObfuscatorJsonConverterTests.WriteHappyJson [FAIL]
  Assert.Equal() Failure
    Expected: 2.5
    Actual:   2,5
  Stack Trace:
    tests\NuGetGallery.Core.Facts\Auditing\ObfuscatorJsonConverterTests.cs(68,0): at NuGetGallery.Auditing.ObfuscatorJsonConverterTests.WriteHappyJson()
```

I'll see if I can submit a PR with a fix.

**Comment by skofman1** [comment]
@khellang , thanks for the investigation. A PR with a fix would be appreciated!

**Comment by khellang** [comment]
I've sent #5662 which should fix both this and #5661 😄

**Comment by skofman1** [comment]
Thanks for the fix! Closing the issue.
